docs/structured-streaming-programming-guide.md

```diff
@@ -2360,8 +2360,35 @@ Here are the configs regarding to RocksDB instance of the state store provider:
     <td>The maximum number of MemTables in RocksDB, both active and immutable. Value of -1 means that RocksDB internal default values will be used</td>
     <td>-1</td>
   </tr>
+  <tr>
+    <td>spark.sql.streaming.stateStore.rocksdb.boundedMemoryUsage</td>
```

Comment:
Wonder if we can add them into `SQLConf` or `StaticSQLConf` so the docs can be generated, and to make it clear which value is the default.

For instance, we automatically generate the docs from SQL conf.

Reply:
Can be done separately in another PR.

Reply:
Maybe better to defer this till we make RocksDB state store provider as default implementation.